comtypes/GUID.py

```diff
@@ -50,19 +53,23 @@ def __unicode__(self):
         result = p.value
         _CoTaskMemFree(p)
         return result
+
     __str__ = __unicode__
 
     def __cmp__(self, other):
         if isinstance(other, GUID):
-            return cmp(binary(self), binary(other))
+            return cmp(binary(self), binary(other))  # NOQA
+
```

Comment:
again code blocks as defined by pep8. a return can be considered the end of a logical section of code and thus a separation should be used.

Reply:
technically that whole method is incorrect as it should read

```python
    def __cmp__(self, other):
        if isinstance(other, GUID):
            res = cmp(binary(self), binary(other))  # NOQA
        else:
            res = -1
        return res
```

but seeing as how the method is slated for removal anyhow it becomes a moot point. I wasn't paying attention when I added the blank line and I should have left it alone because it is going to be removed anyhow.